doc/classes/OS.xml

```diff
@@ -809,7 +809,7 @@
 			<param index="0" name="uri" type="String" />
 			<description>
 				Requests the OS to open a resource identified by [param uri] with the most appropriate program. For example:
-				- [code]OS.shell_open("C:\\Users\name\Downloads")[/code] on Windows opens the file explorer at the user's Downloads folder.
+				- [code]OS.shell_open("C:\\\\Users\\name\\Downloads")[/code] on Windows opens the file explorer at the user's Downloads folder.
```

Comment:
I haven't used this method before but fairly sure there's too many backslashes here at the start:

```suggestion
				- [code]OS.shell_open("C:\\Users\\name\\Downloads")[/code] on Windows opens the file explorer at the user's Downloads folder.
```

Reply:
Indeed, you're absolutely right, @RedMser. It doesn’t even make sense to have so many backslashes. I've corrected this and also added a updated the PR description.